python/hail/environment.yml

```diff
@@ -1,7 +1,7 @@
 name: hail
 dependencies:
 - python=3.6
-- Sphinx=1.5.4
+- Sphinx=1.7.1
```

Comment:
I think we shouldn't specify minor versions here. if we say `1.7` it will get the latest 1.7.x release